docker/Dockerfile-alpine-3.21

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/powershell:alpine-3.20
+FROM mcr.microsoft.com/dotnet/sdk:9.0-alpine3.21
```

Comment:
Is it possible to use .NET 8.0 docker image since 8.0 is LTS? So it should be 8.0--alpine3.21